src/fit.rs

```diff
@@ -207,8 +322,12 @@ pub fn fit_to_cubic(
     let d = end.p - start.p;
     let th = d.atan2();
     let chord2 = d.hypot2();
-    let th0 = start.tangent.atan2() - th;
-    let th1 = th - end.tangent.atan2();
+    fn mod_2pi(th: f64) -> f64 {
```

Comment:
could you use `f64::rem_euclid` for this?

Reply:
It wouldn't be a perfect match, because this is based on rounding, so giving the smallest absolute value, while rem_euclid gives the smallest nonnegative value. Also I checked on Godbolt and at least in some cases my code is faster, as fmod is a function call while rounding can be effected by vroundsd (when the avx level is supported).